app/views/step_by_step_pages/show.html.erb

```diff
@@ -17,20 +17,15 @@
 
 <%= render 'shared/steps/form_notice', message: notice, status: "success" %>
 
-<h1><%= @step_by_step_page.title %></h1>
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
```

Comment:
Is this just changing how the title is rendered?

Reply:
This makes the heading consistent with the other tabs, before it was not noticiable as you were transistioning across multiple pages.

Reply:
So perhaps do this first, separately?